TF_Service_dotNet/TouchFree/InteractionZoneEvent.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Ultraleap.TouchFree.Library
+{
+
+    [Serializable]
+    public struct InteractionZoneEvent
```

Comment:
🤷 Can be shorter declaration as a record
```cs
[Serializable]
public record struct InteractionZoneEvent(InteractionZoneState state);
```

Reply:
Updated to a record